### PR TITLE
[passport-cognito] Add alterantive signature for verifyFunction which includes session obj

### PR DESCRIPTION
**Pull request by bertyhell**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [ ] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [ ] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

File: types/passport-cognito/index.d.ts [hunk 1]
```diff
@@ -21,8 +21,21 @@ export type CognitoVerifyFunction = (
   done: (error: any, user?: any) => void,
 ) => any;
 
+export type CognitoVerifyFunctionWithSession = (
+  accessToken: string,
+  idToken: string,
+  refreshToken: string,
+  user: object,
+	session: {
```
Comment (DangerBotOSS) [inline types/passport-cognito/index.d.ts:29]: - :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.

**Comment by DangerBotOSS** [comment]
Inspecting the JavaScript source for this package found some properties that are not in the .d.ts files.
The check for missing properties isn't always right, so take this list as advice, not a requirement.
## passport-cognito ([<kbd>unpkg</kbd>](https://unpkg.com/browse/passport-cognito@latest/))
was missing the following properties:
1. The declaration doesn't match the JavaScript module 'passport-cognito'. Reason:
The JavaScript module can be called or constructed, but the declaration module cannot.

The most common way to resolve this error is to use 'export =' syntax.
To learn more about 'export =' syntax, see https://www.typescriptlang.org/docs/handbook/modules.html#export--and-import--require.
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against bcb40389d608921c1dd135b545f03602c58ed5d3
</p>

File: types/passport-cognito/index.d.ts [hunk 2]
```diff
@@ -8,22 +8,35 @@ import passport = require('passport');
...
 export interface CognitoStrategyOptions {
-  userPoolId: string;
-  clientId: string;
-  region: string;
+    userPoolId: string;
+    clientId: string;
+    region: string;
 }
 
 export type CognitoVerifyFunction = (
```
Comment (domdomegg) [inline types/passport-cognito/index.d.ts:16]: This can be an interface

File: types/passport-cognito/index.d.ts [hunk 3]
```diff
@@ -8,22 +8,35 @@ import passport = require('passport');
...
-  user: object,
-  done: (error: any, user?: any) => void,
+    accessToken: string,
+    idToken: string,
+    refreshToken: string,
+    user: object,
+    done: (error: any, user?: any) => void,
+) => any;
+
+export type CognitoVerifyFunctionWithSession = (
```
Comment (domdomegg) [inline types/passport-cognito/index.d.ts:24]: This can be an interface, and can extend `CognitoVerifyFunction` to reduce duplication

**Review by domdomegg — APPROVED** [review]
LGTM. Have suggested some minor improvements.

**Pull request closed without merging**